python/grizzly/grizzly/grizzly.py

```diff
@@ -5,6 +5,367 @@
 from lazy_op import LazyOpResult, to_weld_type
 from weld.weldobject import *
 
+class DataFrameWeldLoc:
```

Comment:
Comment on what this is used for would be good

Reply:
Addressed.